applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian.cpp

```diff
@@ -298,30 +299,35 @@ void UpdatedLagrangian::CalculateElementalSystem( LocalSystemComponents& rLocalS
 
     // Set constitutive law flags:
     Flags &ConstitutiveLawOptions=Values.GetOptions();
-    ConstitutiveLawOptions.Set(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN);
-    ConstitutiveLawOptions.Set(ConstitutiveLaw::COMPUTE_STRESS);
     ConstitutiveLawOptions.Set(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR);
 
-    // Auxiliary terms
-    Vector volume_force;
+    if (!rCurrentProcessInfo.GetValue(IS_EXPLICIT))
```

Comment:
warning, `rCurrentProcessInfo` is not `const`, this call will add `IS_EXPLICIT` to the ProcessInfo

=> inside the elements ProcessInfo should only be passed around as `const`